packages/app-extension/src/components/Unlocked/Balances/TokensWidget/Send.tsx

```
@@ -615,16 +649,29 @@ export function useIsValidAddress(
         // The account data has been successfully validated.
         setAddressError(false);
         setAccountValidated(true);
-        setNormalizedAddress(address);
+        setNormalizedAddress(pubkey.toString());
       } else if (blockchain === Blockchain.ETHEREUM) {
         // Ethereum address validation
         let checksumAddress;
-        try {
-          checksumAddress = ethers.utils.getAddress(address);
-        } catch (e) {
-          setAddressError(true);
-          return;
+
+        if (address.includes(".eth")) {
+          try {
+            checksumAddress = await ethereumProvider?.resolveName(address);
```

Similar to above, we should throw if ethereumProvider is undefined outside of this try/catch so it doesn't manifest as a validation error if we forget to pass it